ansible/roles/artifactory/tasks/main.yml

```diff
@@ -0,0 +1,39 @@
+---
+- name: Install Python dependencies
+  pip:
+    name: ['boto', 'boto3', 'botocore']
+    umask: "0022"
+
+- name: Create a directory if it does not exist
```

Comment:
I don't think this is needed. An alternative might be to use tempfile as done [here](https://github.com/companieshouse/sonar-ami/blob/151b7bfb53314214e70d0eae80fb9979dec21b5e/ansible/roles/sonarqube/tasks/setup_sonarqube.yml#L4)